prow/gitee-plugins/reminder/reminder.go

```
@@ -0,0 +1,158 @@
+	}
+	return pluginHelp, nil
+}
+
+func (re *reminder) PluginName() string {
+	return "reminder"
+}
+
+func (re *reminder) NewPluginConfig() plugins.PluginConfig {
+	return &configuration{}
+}
+
+func (re *reminder) RegisterEventHandler(p plugins.Plugins) {
+	name := re.PluginName()
+	p.RegisterIssueHandler(name, re.handleIssueEvent)
+	p.RegisterNoteEventHandler(name, re.handleNoteEvent)
+}
+
+func (re *reminder) handleIssueEvent(e *sdk.IssueEvent, log *logrus.Entry) error {
+	funcStart := time.Now()
+	defer func() {
+		log.WithField("duration", time.Since(funcStart).String()).Debug("Completed handleIssueEvent")
+	}()
+
+	if *(e.Action) != "open" {
+		log.Debug("Event is not a creation of a issue, skipping.")
+		return nil
+	}
+	HELPLabel := "stat/help-wanted"
+	issue := e.Issue
+	org := e.Repository.Namespace
+	repo := e.Repository.Path
+	labels := issue.Labels
+	issueNumber := issue.Number
+
+	hasHELP := false
+
+	for _, label := range labels {
+		if !hasHELP && label.Name == HELPLabel {
+			hasHELP = true
+			break
+		}
+	}
+
+	if !hasHELP {
+		if err := re.ghc.CreateComment(org, repo, issueNumber, "Please add labels, for example, "+
+			`if you are filing a runtime issue, you can type "//comp/runtime" in comment,`+
+			` also you can visit "https://shimo.im/sheets/8pKDkqKqdycHRwWV/MODOC/" to find more labels`); err != nil {
+			log.WithError(err).Warningf("Could not add label.")
+		}
+	}
+	return re.ghc.AddLabel(org, repo, issueNumber, HELPLabel)
+}
+
+func (re *reminder) handleNoteEvent(e *sdk.NoteEvent, log *logrus.Entry) error {
+	funcStart := time.Now()
+	defer func() {
+		log.WithField("duration", time.Since(funcStart).String()).Debug("Completed handleNoteEvent")
+	}()
+
+	if *(e.Action) != "comment" {
+		log.Debug("Event is not a creation of a comment, skipping.")
+		return nil
+	}
+
+	userName := e.Comment.User.Name
+	botName, _ := re.ghc.getBotName()
```

It is recommended not to ignore the error, it will affect the subsequent logical judgment